docs/manual/java/guide/cluster/index.toc

```diff
@@ -1,7 +1,8 @@
-PersistentEntity:Persistent Entity
-PersistentEntityCassandra:Cassandra Persistent Entities
-PersistentEntityRDBMS:Relational Database Persistent Entities
-ReadSide:Persistent Read-Side
+PersistentEntity:Persistent Entity;next=ChoosingADatabase,PersistentEntityCassandra,PersistentEntityRDBMS
+ChoosingADatabase:Choosing A Database
+PersistentEntityCassandra:Cassandra Setup
+PersistentEntityRDBMS:Relational Database Setup
+ReadSide:Persistent Read-Side;next=ReadSideCassandra,ReadSideRDBMS
```

Comment:
In #2399 I forgot to rename the pages in the index:

```
PersistentEntityCassandra:Cassandra Setup
PersistentEntityRDBMS:Relational Database Setup
```

Reply:
I've also started overwriting the default behavior of the index to provide a more meaningful navigation (see the addition of all the `;next` metadata above). This is is in preparation of #2405 .